### PR TITLE
feat: wasmcloud 1.0 updates

### PR DESCRIPTION
**Pull request by brooksmtownsend**
## Feature or Problem
This PR updates our quickstart and related documentation to work with the 1.0 release candidate that we're releasing today. It includes instructions for installing wash 0.27.0-alpha.2 and fixes the Rust component example.

We still need to update the project template for the Python, Go, and TypeScript examples. I'll take that on first thing tomorrow.

### CODE REVIEW

File: docs/tour/hello-world.mdx [hunk 1]
```diff
@@ -383,72 +384,51 @@ Capabilities are _deny by default_, the link inside of `wadm.yaml` explicitly al
...
   components:
-    # Your application component, started from the wasmCloud example OCI artifact
-    - name: hello
+    - name: http-component
       type: component
       properties:
-        image: wasmcloud.azurecr.io/hello:0.1.7
+        # Your manifest will point to the path of the built component, you can also
+        # start published components from OCI registries
+        image: wasmcloud.azurecr.io/http-hello-world:0.1.0
```
Comment (vados-cosmonic) [inline docs/tour/hello-world.mdx:407]: An example of the relative `file://` setting might be nice here

File: docs/tour/scale-and-distribute.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+sidebar_position: 3
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import washboard_hello from '../images/washboard_hello.png';
+
+## Scaling up 📈
+
+WebAssembly can be easily scaled due to its small size, portability, and [wasmtime](https://wasmtime.dev/)'s ability to efficiently instantiate multiple instances of a single WebAssembly component. We leverage these aspects to make it simple to scale your applications with wasmCloud. Components only use resources when they're actively processing requests, so you can specify the number of replicas you want to run and wasmCloud will automatically scale up and down to meet demand. Let's scale up our hello world application to 50 replicas by editing `wadm.yaml`:
```
Comment (vados-cosmonic) [inline docs/tour/scale-and-distribute.mdx:12]:
```suggestion
wasmCloud's efficiency and scalability can be attributed to two factors:
- WebAssembly's small size and portability
- [`wasmtime`](https://wasmtime.dev)'s efficient instantiation of multiple instances of a single WebAssembly component

By using the WebAssembly ecosystem, wasmCloud makes it easy to scale your applications. 

Components only use resources when they're actively processing requests, so you can specify the number of replicas you want to run and wasmCloud will automatically scale up and down to meet demand. 

Let's scale up our hello world application to 50 replicas by editing `wadm.yaml`:
```

**Review by lachieh — APPROVED** [review]

File: docs/tour/hello-world.mdx [hunk 3]
```diff
@@ -157,36 +143,42 @@ This component is a simple Python project with a few extra goodies included for
...
-            formId='71e74f55-cc30-41de-9d41-e3d9dc159c71'
-            onSubmit={() => console.log('Submitted form')}
-            onReady={(form) => console.log('Form ready for submit')}
-            region="na1"
-            loading={<div>Loading...</div>}
-        />
-    </div>
+If you prefer working in a language that isn't listed here, let us know!
+
+{' '}
```
Comment (vados-cosmonic) [inline docs/tour/hello-world.mdx:158]: is this to ensure there's some spacing?

File: docs/tour/hello-world.mdx [hunk 4]
```diff
@@ -203,6 +196,15 @@ Component built and signed and can be found at "/Users/wasmcloud/hello/build/htt
 
 We'll use `wash` and [wadm](/docs/category/declarative-application-deployment-wadm), the wasmCloud application deployment manager, to start this component, the httpserver capability, and link them together to configure them.
 
+{/* TODO(brooksmtownsend): Remove this after wadm supports config */}
```
Comment (vados-cosmonic) [inline docs/tour/hello-world.mdx:199]: Maybe it's better to make an issue for this, and tie it to an issue that tracks config support in wadm if there is one?

**Review by vados-cosmonic — APPROVED** [review]
LGTM :rocket:

**Review by ericgregory — APPROVED** [review]

File: docs/tour/hello-world.mdx (hunk 4 above)
Comment (brooksmtownsend) [reply docs/tour/hello-world.mdx:199]: Definitely!

**Pull request merged**